federation/cmd/federation-controller-manager/app/controllermanager.go

```diff
@@ -144,5 +145,11 @@ func StartControllers(s *options.CMServer, restClientCfg *restclient.Config) err
 	if err := servicecontroller.Run(s.ConcurrentServiceSyncs, wait.NeverStop); err != nil {
 		glog.Errorf("Failed to start service controller: %v", err)
 	}
+
+	nsClientset := federationclientset.NewForConfigOrDie(restclient.AddUserAgent(restClientCfg, "namespace-controller"))
+	namespaceController := namespacecontroller.NewNamespaceController(nsClientset)
+	namespaceStopChannel := make(chan struct{})
+	namespaceController.Run(namespaceStopChannel)
```

Comment:
nit: we are using wait.NeverStop everywhere, so we probably also want it here (no need to create namespaceStopChannel)